scss/_orange-navbar.scss

```diff
@@ -1,63 +1,40 @@
 // Contents
 //
 // Supra bar
-
-
-// Supra bar
-//
 // Brand exclusive thin navbar that can be used with orange navbar
 .supra {
-  display: none;
-  padding-top: 0;
-  padding-bottom: 0;
-  border-bottom: $navbar-orange-supra-border-bottom-width solid $gray-700;
-
-  .container-xxl {
-    padding-right: $navbar-orange-supra-padding-x;
-    padding-left: $navbar-orange-supra-padding-x;
-  }
-
-  @include media-breakpoint-up(md) {
-    display: flex;
-  }
-
-  .navbar-nav .nav-link {
-    padding: $navbar-orange-supra-link-padding-top $navbar-orange-supra-link-padding-x $navbar-orange-supra-link-padding-bottom;
-    font-size: $font-size-sm;
-    line-height: $line-height-sm;
+  --#{$variable-prefix}navbar-padding-y: 0px; /* stylelint-disable-line length-zero-no-unit */
```

Comment:
I tried many values here, the fact is that `calc` doesn't allow multiplication between 0 and a number without unit. I didn't find any to bypass properly, but if anyone has an idea, it would be great indeed 😃